ios/FluentUI/Shimmer/ShimmerTokenSet.swift

```diff
@@ -82,6 +85,14 @@ public class ShimmerTokenSet: ControlTokenSet<ShimmerTokenSet.Tokens> {
                     }
                 }
 
+            case .viewTint:
+                return .dynamicColor {
+                    return DynamicColor(light: ColorValue(0xF1F1F1) /* gray50 */,
+                                            lightHighContrast: ColorValue(0x919191) /* gray400 */,
```

Comment:
are these comments meant to be here?

Reply:
that's how i saw it done elsewhere in the repo (e.g. ResizingHandleTokenSet and TabBarItemTokenSet)